docs_theme/overview.md

```diff
@@ -1 +1,18 @@
-Overview of the library here
+|docs|[![docs](https://github.com/FedericoTartarini/jsthermalcomfort/actions/workflows/test_docs.yml/badge.svg)][docs_link]|
+|:---:|:---|
+|**build**|[![build](https://github.com/FedericoTartarini/jsthermalcomfort/actions/workflows/build.yml/badge.svg)][build_link]|
+|**tests**|[![test](https://github.com/FedericoTartarini/jsthermalcomfort/actions/workflows/test.yml/badge.svg)][test_link]|
+|**snyk security**|[![snyk security](https://github.com/FedericoTartarini/jsthermalcomfort/actions/workflows/snyk.yml/badge.svg)][snyk_link]|
+|**license**|[![license](https://custom-icon-badges.demolab.com/github/license/denvercoder1/custom-icon-badges?logo=law)](https://github.com/CenterForTheBuiltEnvironment/pythermalcomfort/blob/master/LICENSE)|
+
+
+Package to calculate several thermal comfort indices (e.g. PMV, PPD, SET, adaptive) and convert physical variables.  
+
+Please cite us if you use this package: [Tartarini, F., Schiavon, S., 2020. pythermalcomfort: A Python package for thermal comfort research. SoftwareX 12, 100578. https://doi.org/10.1016/j.softx.2020.100578](https://www.sciencedirect.com/science/article/pii/S2352711020302910) 
+
+- Free software: MIT license
+
+[docs_link]:https://github.com/FedericoTartarini/jsthermalcomfort/actions/workflows/test_docs.yml
+[build_link]:https://github.com/FedericoTartarini/jsthermalcomfort/actions/workflows/build.yml
+[test_link]:https://github.com/FedericoTartarini/jsthermalcomfort/actions/workflows/test.yml
+[snyk_link]:https://github.com/FedericoTartarini/jsthermalcomfort/actions/workflows/snyk.yml
```

Comment:
what are these for?

Reply:
Clicking those badges will lead us to the corresponding link of the actions in the repo

Reply:
does it work correctly?